code/game/machinery/camera/camera.dm

```diff
@@ -328,10 +328,11 @@
 			itemname = pressed_pda.name
 			info = pressed_pda.notehtml
 
-		var/sanitized_name = sanitize(itemname)
+		itemname = sanitize(itemname)
+		info = sanitize(info)
 
```

Comment:
This will break the html window. Maybe best to do separately

Reply:
Yeah, tbf info can only really go bad if an admin is varediting it, which makes it pretty mild for XSS.
I'll strip this

Reply:
yeah if an admin wants to xss they can just xss